README.md

```diff
@@ -48,6 +48,6 @@ aio config:set aio-cli-plugin-commerce-admin <path_to_json_file>
 
 ```
 aio commerce-gateway:tenant:create BODYJSONFILE
-aio commerce-gateway:tenant:update tenantid BODYJSONFILE
-aio commerce-gateway:tenant:get tenantid
+aio commerce-gateway:tenant:update meshId BODYJSONFILE
+aio commerce-gateway:tenant:get meshId
 ```
```

Comment:
Are we not adding the new feature you introduced to write the mesh to a file ?

Reply:
Yeah, forgot to update these. Thanks guys. Gonna take care of this.